cmd/modern/root/config/add-user.go

```diff
@@ -30,13 +30,21 @@ func (c *AddUser) DefineCommand(...cmdparser.CommandOptions) {
 		Short: "Add a user",
 		Examples: []cmdparser.ExampleOptions{
 			{
-				Description: "Add a user",
+				Description: "Add a user (using the SQLCMD_PASSWORD environment variable)",
 				Steps: []string{
 					fmt.Sprintf(`%s SQLCMD_PASSWORD=<password>`, pal.CreateEnvVarKeyword()),
```

Comment:
>password [](http://example.com/codeflow?start=38&length=8)

is credscan smart enough to ignore this or do we need to use <placeholderpassword> instead?

Reply:
I've replaced all usage with: <placeholderpassword>